raster-layer-file/README.md

```diff
@@ -0,0 +1,37 @@
+# Raster Layer File
+
+Demonstrates how to create and use a raster layer made from a local raster file.
+
+![](raster-layer-file.png)
```

Comment:
@TADraeseke Fair point, I updated the screenshot to match image dimensions (700x420).